guide/english/css/css3-media-queries/index.md

```diff
@@ -27,7 +22,7 @@ The thought process should be:
     }
 ```
 
-The `p` tag will have a padding of 30px as soon as the screen reaches min 768px width.</p>
+This media query adds 30px of padding to all `p` tags when the screen is wider than 767px.
```

Comment:
This line could be interpreted wrongly because:
`...adds 30px of...` would imply that when the screen width hits `768px`, `30px` padding will be <em>ADDED</em>  which is fundamentally wrong as actually the padding would be changed to `30px`.

for example,  what your changed line implies is that if original padding
is `10px` and when media-query is successful, then the padding would become `(10px + 30px)` which will be wrong.
The original line was more descriptive towards the actual working. Consider changing this line

Reply:
Oh, I see how it could be interpreted that way. Will edit closer to original wording.